stock_product_pack/tests/test_stock_product_pack.py

```diff
@@ -122,10 +81,12 @@ def test_compute_quantities_dict(self):
             }
         )
         picking.action_confirm()
-        self.assertEqual(self.pack_dc.virtual_available, 5)
-        self.assertEqual(self.pack_dc.qty_available, 0)
+        pack = self.pack_dc.with_context()
+        self.assertEqual(pack.virtual_available, 5)
+        self.assertEqual(pack.qty_available, 0)
```

Comment:
Just leave as it was
```suggestion
        self.assertEqual(self.pack_dc.virtual_available, 5)
        self.assertEqual(self.pack_dc.qty_available, 0)
```

Reply:
Of course :+1: . Making them now

Reply:
Changes made